controllers/users.js

```diff
@@ -35,13 +35,7 @@ function update(req, res, next) {
 }
 
 function updateProfile(req, res, next) {
-  const {
-    name, email, password, geolocation,
-  } = req.body
-
-  req.user.set({
-    name, email, password, geolocation,
-  }).save()
+  req.user.set(req.body).save()
```

Comment:
I think it's a better idea to destructure `req.body` into separate variables, because there might be a chance where `req.body` contains a property that shouldn't be in a user document.

Reply:
we should not worry about req.body contains a property that is not included in a user document because User.set only update properties that is included in the schema and ignores unnecessary properties that is not present in the define schema.

Reply:
Neat! I checked and you're right. TIL. I stand corrected.